src/content/app/entity-viewer/variant-view/transcript-consequences/TranscriptConsequences.tsx

```diff
@@ -44,6 +47,12 @@ type Props = {
   activeAlleleId: string;
 };
 
+type transcriptConsequencesWithTranscriptType = Array<
```

Comment:
Style.
- Begin type names with capital letter
- Don't need the word `type` in the end of the name (unless there is a clash with the name of a variable)
- When you declare types, describe the type of a single member of an array (i.e. type of a transcript consequence with transcript, not the type of the whole array). Then, when you use this type, you can just mark that you are using it as an array (i.e. instead of `const foo = (things: Things) => { ... }`, prefer `const foo = (things: Thing[]) => { ... }`).

Reply:
There is another word `transcriptConsequencesWithTranscript` so adding the suffix is better.